values.yaml

```diff
@@ -107,6 +118,13 @@ autoscaling:
   targetCPUUtilizationPercentage: 80
   targetMemoryUtilizationPercentage: 80
 
+# Adds topology spread constraints to the relay deployment
```

Comment:
Similar request here. I think we can nest this under `pod`.

Reply:
Thanks, has been moved.